src/main/kotlin/frc/robot/subsystems/DriveSubsystem.kt

```diff
@@ -6,7 +6,18 @@ import frc.robot.Mockable
 
 @Mockable
 class DriveSubsystem(private val drive: DifferentialDrive) : SubsystemBase() {
+    private var power = Pair(0.0, 0.0)
+
     fun arcadeDrive(fwd: Double, rot: Double) {
+        power = Pair(fwd, rot)
+    }
+
+    override fun periodic() {
+        val (fwd, rot) = power
```

Comment:
Did you see this pattern used elsewhere? It looks like [this sample drive subsystem](https://docs.wpilib.org/en/latest/docs/software/examples-tutorials/trajectory-tutorial/creating-drive-subsystem.html) just has a drive method that directly calls arcade drive. I worry that using a variable to store the last power provided is working around an important safety measure which is that if the driver station, network, or robot code is lagging behind such that it would appear unresponsive, its default is to stop the robot rather than accidentally/potentially running into things.